bin/dig.bat

```diff
@@ -0,0 +1,2 @@
+@ECHO OFF
+"%~dp0%~n0\%~n0.exe" %*
```

Comment:
The `"%~dp0%~n0\%~n0.exe"` can also be written as `"%~dpn0\%~n0.exe"`, which seems less cryptic for me. The other thing is `@ECHO OFF` is propagated to subprocesses, which makes no difference here, but just simple `@"%~dpn0\%~n0.exe" %*` is optimal imho.